tools/autograph-monitor/constants.go

```diff
@@ -1,18 +1,100 @@
+var firefoxPkiStageRootHashes = []string{
+	`C0:F0:5D:59:B1:FD:E2:57:80:85:4C:32:FA:E8:FA:BA:84:81:C2:33:B4:C1:D3:90:CC:A5:F2:CE:A8:19:30:EE`,
+	`45:C3:7F:3A:09:A6:D7:0E:0F:A3:21:FB:29:75:3B:A7:99:8F:12:59:B3:27:72:76:8F:23:CC:DC:24:83:67:98`,
+	`DB:74:CE:58:E4:F9:D0:9E:E0:42:36:BE:6C:C5:C4:F6:6A:E7:74:7D:C0:21:42:7A:03:BC:2F:57:0C:8B:9B:90`,
+}
 
-// firefoxPkiStageRoot is the CA root cert for the Addon stage code
+// firefoxPkiStageRoots is the list of CA root certs for the Addon stage code
 // signing PKI
-const firefoxPkiStageRoot = `-----BEGIN CERTIFICATE-----
+var firefoxPkiStageRoots = []string{
+	// new root cert
+	`-----BEGIN CERTIFICATE-----
+MIIGRDCCBCygAwIBAgIBATANBgkqhkiG9w0BAQwFADBzMQswCQYDVQQGEwJVUzEc
+MBoGA1UECgwTTW96aWxsYSBDb3Jwb3JhdGlvbjEoMCYGA1UECwwfTW96aWxsYSBT
+dGFnaW5nIFNpZ25pbmcgU2VydmljZTEcMBoGA1UEAwwTY2FzLXJvb3QtY2Etc3Rh
+Z2luZzAgFw0yNDAyMTIwMDAwMDBaGA8yMDUwMTIzMTAwMDAwMFowczELMAkGA1UE
+BhMCVVMxHDAaBgNVBAoME01vemlsbGEgQ29ycG9yYXRpb24xKDAmBgNVBAsMH01v
+emlsbGEgU3RhZ2luZyBTaWduaW5nIFNlcnZpY2UxHDAaBgNVBAMME2Nhcy1yb290
+LWNhLXN0YWdpbmcwggIiMA0GCSqGSIb3DQEBAQUAA4ICDwAwggIKAoICAQDJLI7O
+heP24t+vR/afP2U80Os1UQIVWmZLd4Z9/ybt9NaXzeCqCd5uV7lpNCvYZLjw57xN
+0ZrQiuTw1lAf5IgLKFkjWIWU6YbagQKxX8zGzZ1t8IpvXlP5Qo0kPkYHyJ6FB1GW
+JKiHlj0tqDpvfb7oooImsdRvw2lhvXa9m8ttFnUS69Wf9Xh14hSb+Cqg5HfHxBkB
+Z/UcrYeOlq5fkxkrTjrCa+XH2942+ZzcWHSjOF3Sf2/gMug1Dc01ORVbloz5e7AI
+fdA9Ng9DSjUffCaiTvYqz8f8wZ3gYTd6R8oFfo5Y8y2hjPxPHsTsYaiVvwdx8VBZ
+1CBvAGbK4PQHPa4nboJhIuL1JDUhZJAOy/4i2pzn77ce1399CrSLfBzGc6ZWmJ4a
+aoG2wdybiGahMasAYf8M+EWlhcnr0YUjnHDWZ50jJ8D62O+SLMRLyWYmZRfYhOHf
+3Ed4db1tO2xGDrwoSncbBt706zyBZcNwd1lC21j+cjIDHEmnvv22Csq9KKqMbyd6
+v76Ils02uWl63EWlXVbnbNrWW2u+7YvrxZ5DK/GZZIorFZuOzoaFH4VUZ5waxofi
+tCN2C7W/QiejqnXUPZQgNQU48aOq3JlvoiwvVFxoejmfw/6EiAMyuE6Tx2odKFBS
+1+3EKHpn/AWc2yAos3NfFUhDo3vxonuYwbSlxwIDAQABo4HgMIHdMAwGA1UdEwQF
+MAMBAf8wDgYDVR0PAQH/BAQDAgEGMBYGA1UdJQEB/wQMMAoGCCsGAQUFBwMDMB0G
+A1UdDgQWBBRnHYjw7dUlrInD0HrEGq1KNJYA9jCBhQYDVR0jBH4wfKF3pHUwczEL
+MAkGA1UEBhMCVVMxHDAaBgNVBAoME01vemlsbGEgQ29ycG9yYXRpb24xKDAmBgNV
+BAsMH01vemlsbGEgU3RhZ2luZyBTaWduaW5nIFNlcnZpY2UxHDAaBgNVBAMME2Nh
+cy1yb290LWNhLXN0YWdpbmeCAQEwDQYJKoZIhvcNAQEMBQADggIBABOMi2kAbJRG
+kvGjSeeQXKiXNO/dfLadYG938kl/Xagc7eGWvccHeTPBT7lkiQQko9ucXe4pMzOD
+YGyBZyeMc3uFFqjsT9gg/r4KyitxE987qqPS0cjG1up72UyhfNUBgBxCCx0QrRpZ
+AbuOAuoTg9tE2uLdqBIemWHHBl/DXKYpmfOd9d6DPI1zxxLp/EegC/mEmiCysA20
+wBy/dbK6uhQ+gjqaImjclejaAg07b2HR1HiXNNDdCqL3KE///stTkFCrTq9CFU74
+ZdAiCEPSDYU2Qag/jUgzxqNrsPNjA9CkseCezmaNiU28zuEX8xbbhKxoyagAA4mS
+lVza5TRvfBo2jbnBFpgfChplDTTfK653k5MF6Q5rPiz4guNqylzq9ZqxGbPGKQbz
+0K7PqwKAQTWQ/lo85Q8MFVOSgFeMNOYv6ak+N3QXEpzEtMs1Ta5i9Pla4ia+ZZi8
+4lhd/mblxrq8QcpQmrHgXjb/Kpm5BbKnJYUIYidEfW1BMktsPZVbDc0ePa8hu6oZ
+Pafl5Rch9rAxeaMFOgYMeOJwlRCt2Fbm+mtnWKFdxy9Peib75pXQD43qYbnK8ucj
+xNCR3VBsgIgAhyO11hXiBo5nX5OHn2PG1MPV9+H0RQb6g5zOuKrWoh9CzB8mBl7d
+0JMkzy6TFAcAo3KL5ZNvRqqB8xRmIYLK
+-----END CERTIFICATE-----`,
+	// current root cert
```

Comment:
"current" isn't accurate. I know we called it "cas_cur" but that was a temporary thing. It's now the one that _isn't_ used by staging services. `cas_new` is. I know, this is confusing now, but we had to optimize for not confusing the folks doing backwards compat testing during cert succession.

Future signers won't be prefixed with `cas_new_` but will use that "cas_new" intermediate cert.

Reply:
Yeah, updating comments to make this (hopefully) a little less confusing.